Firmware/Firmware.ATXMEGA/SoundCard/app_ios_and_regs.h

```diff
@@ -132,34 +132,18 @@ typedef struct
 	uint8_t REG_DIGITAL_INPUTS;
 	uint8_t REG_DI0_CONF;
 	uint8_t REG_DI1_CONF;
-	uint8_t REG_DI2_CONF;
-	uint8_t REG_DI0_SOUND_INDEX;
-	uint8_t REG_DI1_SOUND_INDEX;
-	uint8_t REG_DI2_SOUND_INDEX;
-	uint16_t REG_DI0_FREQ;
-	uint16_t REG_DI1_FREQ;
-	uint16_t REG_DI2_FREQ;
+	uint16_t REG_DI0_SOUND_INDEX;
+	uint16_t REG_DI1_SOUND_INDEX;
 	uint16_t REG_DI0_ATTNUATION_LEFT;
```

Comment:
Typos

Reply:
There is another typo right below it too.